modules/rostests/apitests/shell32/ItemIDList.cpp

```diff
@@ -0,0 +1,149 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Test for SHSimpleIDListFromPath
+ * COPYRIGHT:   Copyright 2024 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
```

Comment:
I think you can add your copyright line @whindsaks since you quite extended the tests.